lib/components/EstateCard/estate_text.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+class EstateText extends StatefulWidget {
+  final String text;
+  const EstateText({Key? key, required this.text}) : super(key: key);
+
+  @override
+  State<EstateText> createState() => _EstateTextState();
+}
+
+class _EstateTextState extends State<EstateText> {
```

Comment:
Não vejo necessidade em componentizar um widget de texto apenas. Eu entendi a ideia do porquê fizeram isso, mas acredito que um componente só para isso seja demais.
Uma ideia do que podem fazer é criar uma variável dentro da pasta Utils ( pessoal do cadastro criou ) para armazenar esse Text Style, algo como:
```suggestion
static final TextStyle estateTextStyle = TextStyle(
    color: Colors.black,
    fontSize: 20,
    fontFamily: 'Roboto-Regular',
  );
```